### ISSUE TITLE
v2.38.0 just spits error in a loop

### ISSUE THREAD

**Issue by gus4rs** [ledgerwatch/erigon#6781]
```
Feb 05 21:56:21 localhost.localdomain erigon[4041988]: [INFO] [02-05|21:56:21.514] [2/15 Headers] Waiting for Consensus Layer...
Feb 05 21:56:21 localhost.localdomain erigon[4041988]: [EROR] [02-05|21:56:21.514] Staged Sync                              err="send on closed channel, trace: [stageloop.go:133 panic.go:890 chan.go:206 chan.go:145 request_list.go:157 request_list.go:125 stage_headers.go:177 stage_headers.go:142 default_stages.go:35 sync.go:353 sync.go:255 stageloop.go:167 stageloop.go:95 asm_amd64.s:1594]"
```

over and over

**Comment by abhishektvz** [comment]
could it be due to eth68 being enabled and not yet being suppported by external CLs, from what I understand eth68 requires certain functionality on the consensus clients as well

can you try to run v2.38.0 by disabling eth68,
`--p2p.protocol 66,67`

**Comment by AskAlexSharov** [comment]
fixing by https://github.com/ledgerwatch/erigon/pull/6763

**Comment by greenertux** [comment]
Fixed by v3.38.1 for me
